src/utils/pickItem/pickImage.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { ContextValueQuickPickStep, runQuickPickWizard, type AzureWizardPromptStep, type IActionContext, type QuickPickWizardContext } from "@microsoft/vscode-azext-utils";
+import { ext } from "../../extensionVariables";
+import { type ContainerAppItem } from "../../tree/ContainerAppItem";
+import { ImageItem } from "../../tree/containers/ImageItem";
+import { getPickContainerSteps } from "./pickContainer";
+import { pickContainerApp } from "./pickContainerApp";
+import { type RevisionDraftPickItemOptions } from "./PickItemOptions";
+
+function getPickImageStep(): AzureWizardPromptStep<QuickPickWizardContext> {
+    return new ContextValueQuickPickStep(ext.rgApiV2.resources.azureResourceTreeDataProvider, {
+        contextValueFilter: { include: ImageItem.contextValueRegExp },
+        skipIfOne: true,
+    });
+}
+
+export async function pickImage(context: IActionContext, options?: RevisionDraftPickItemOptions): Promise<ImageItem> {
+    const containerAppItem: ContainerAppItem = await pickContainerApp(context);
```

Comment:
What do you think about leveraging the `getPickContainerAppSteps` instead and including it in the prompt steps here? The fact that this would end up being two separate wizards may be a bit jarring plus it means a user can't go back if they accidentally selected the wrong container.

Reply:
This is a good point, I'll do this instead